jenkins-pipelines/longevity-sla-system-24h.jenkinsfile

```diff
@@ -0,0 +1,12 @@
+#!groovy
```

Comment:
Please move this pipeline under new folder called "sla" or better "workload-prioritization"

Reply:
done